server/container/guice/guice-common/sample-configuration/james-database.properties

```diff
@@ -0,0 +1,40 @@
+#  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Here I think we need to : 

 - move sample-configuration to cassandra-guice
 - create one sample-configuration for jpa-guice

Otherwise it is confusing, we start mixing JPA and cassandra configuration. We need clear, separated sample-configurations.